.github/workflows/prerelease.yml

```diff
@@ -88,11 +88,24 @@ jobs:
     needs: [validate, test-nix, snyk, test-integration-nix]
     steps:
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
       - name: Login to DockerHub
         uses: docker/login-action@v1
         with:
           username: ${{ secrets.OHAI_DOCKER_HUB_ID }}
           password: ${{ secrets.OHAI_DOCKER_HUB_PASSWORD }}
+      - name: Create CHANGELOG.md
+        uses: heinrichreimer/github-changelog-generator-action@v2.2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          issues: true
+          issuesWoLabels: true
+          pullRequests: true
+          prWoLabels: true
```

Comment:
These four shouldn't need to be specified. A bug has been reported upstream: https://github.com/heinrichreimer/action-github-changelog-generator/issues/23